### ISSUE TITLE
🐛 Bug Report: Questions being repeated during every round

### ISSUE THREAD

**Issue by VoidCupboard** [appwrite/demo-quiz-with-flutter#28]
### 👟 Reproduction steps

Start using the app and play the round, you will notice that sometimes the questions asked before are being repeated.

### 👍 Expected behavior

It should not repeat the question

### 👎 Actual Behavior

It actually repeats the question

### 🎲 Appwrite version

Different version (specify in environment)

### 💻 Operating system

Something else

### 🧱 Your Environment

Flutter version: 3.3.4
Dart version: 2.18.2 (stable) (Tue Sep 27 13:24:11 2022 +0200) on "linux_x64"
Appwrite package version: 8.1.0

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by literalEval** [comment]
I would like to work on this issue.

**Comment by SagittariusA11** [comment]
I would like to work on this issue.

**Comment by stnguyen90** [comment]
FYI, I think the problem here is a duplicate question: 

https://github.com/appwrite/demo-quiz-with-flutter/blob/f0555dace658f4e312217d931d01becfd481fc8c/quiz_app_questions.json#L32-L41

**Comment by SagittariusA11** [comment]
Yes, while going through the repo I also found this. So should I just delete the question or add some other question? I would prefer the later.

**Comment by SagittariusA11** [comment]
Have replaced the question and created a pull request.
https://github.com/appwrite/demo-quiz-with-flutter/pull/32

**Comment by stnguyen90** [comment]
Assigning to @SagittariusA11 since there was no response from @literalEval.

**Comment by literalEval** [comment]
> @literalEval, are you still interested in working on this?

Yes I am. Sorry for being late.

**Comment by stnguyen90** [comment]
@literalEval, sorry, I already assigned it to @SagittariusA11. Feel free to try and find another issue
